src/main.rs

```diff
@@ -37,9 +46,25 @@ fn main() {
     if let Some(ref matches) = matches.subcommand_matches("serve") {
         let debug = matches.is_present("debug");
         println!("This doesn't work yet, but it will eventually serve the following database: {} \
-                  on port: {}.  Debugging={}",
+                  on port: {}.",
                  matches.value_of("database").unwrap(),
-                 matches.value_of("port").unwrap(),
-                 debug);
+                 matches.value_of("port").unwrap());
+
+        // set up logging
+        let log_level = if debug {
```

Comment:
Is this formatted by `rustfmt`?  I wonder if this should be all one line, but I'm happy to roll with your preference.